### DISCUSSION TITLE
Release 1.22

### DISCUSSION THREAD

**Discussion by webspiderteam** [webspiderteam/Bluetooth-DMM-For-Windows#28] [Announcements]
- Added ability to Check Updates over Github
- Changed Settings Window
* This version Probably latest version App will update only for fixing issues thats why I added Check Update ability.

<hr /><em>This discussion was created from the release <a href='https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFRelease1.22'>Release 1.22</a>.</em>

**Comment by ludwich66** [comment]
Morning Engin,

i startet a new MQTT Test now.

I do actualize Wiki-Page next Day's.
I hope it is stable now ;-)

You built the new settings page, I think that my icons for the first menu harmonize much better with your new menu than the colorful icons.
But I noticed that honey colors are different in the two menus?

Otherwise everything looks great!
Good work +++

L

**Reply by webspiderteam** [reply]
> I hope it is stable now ;-)

I didn't changed main functions it should stable but who knows :)

> I think that my icons for the first menu harmonize much better with your new menu than the colorful icons.
> But I noticed that honey colors are different in the two menus?

Probably you are right but it look so monochrome for me I just wanted to add some color. 



> Good work +++

thank you :)

**Comment by ludwich66** [comment]
It's Time for a new Youtube-Video ;-)

**Comment by ludwich66** [comment]
Hi Engin,

please look at actual wiki page.
Ideas, suggestions, errors??

You could also put the icon set switchability as an option in the "view settings" ;-)
Now i put a little hint in the Wiki.

L

**Reply by webspiderteam** [reply]
> You could also put the icon set switchability as an option in the "view settings" ;-)

No 😉 it's an Easter egg 😂 


> Now i put a little hint in the Wiki.

I made it hidden a little ;)

**Reply by webspiderteam** [reply]
> please look at actual wiki page.
> Ideas, suggestions, errors??

Thank you very much :) It looks nice but items needs some explaining of what they are. we should explain what they are doing ;) 

BTW why are you always writing it as Menu'e' :) I think its spelling like this in German :)

**Reply by ludwich66** [reply]
Oh, thanks for the Menu hint.

Did you fixed it?
I changed Structure Picture -- you can find master file under "Extras"

yes we can add more Description - it's the first draft ;-)

L

**Reply by webspiderteam** [reply]
> Did you fixed it?

yes I fixed if i didnt missed one :)


> I changed Structure Picture -- you can find master file under "Extras"

I saw it.

**Comment by webspiderteam** [comment]
![FullDisplay](https://user-images.githubusercontent.com/85828505/189528425-f795c589-2f0e-4df3-a913-f3a3d9fbc0da.png)
full display

**Reply by ludwich66** [reply]
?? There is no AC ??

**Reply by webspiderteam** [reply]
Its not sepereted AC/DC is same area as symbol

**Comment by ludwich66** [comment]
Oh Thanks,

crazy overlap ;-)

L

**Reply by webspiderteam** [reply]
> crazy overlap ;-)

actually its not overlapping in using time :)
